src/components/NoteStats/ZapButton.tsx

```diff
@@ -32,9 +32,11 @@ export default function ZapButton({ event }: { event: Event }) {
   const timerRef = useRef<ReturnType<typeof setTimeout> | null>(null)
   const isLongPressRef = useRef(false)
 
+  const { profile: ownProfile } = useNostr()
```

Comment:
Replace `profile` with `pubkey` or `account.pubkey` because the update of the `profile` is not very timely